src/plugins/intel_gpu/src/graph/include/primitive_inst.h

```diff
@@ -195,14 +199,15 @@ class primitive_inst {
     const std::unordered_map<size_t, std::tuple<int64_t, size_t>>& get_profiling_data() const { return _profiling_data; }
     const std::unordered_map<size_t, instrumentation::perf_counter_key>& get_profiling_info() const { return _profiling_info; }
 
-    layout get_node_output_layout() const { return _node_output_layout; }
+    layout get_node_input_layout() const { return _impl_params->get_input_layout(0); }
+    layout get_node_output_layout() const { return _impl_params->output_layout; }
+    std::vector<cldnn::fused_primitive_desc_onednn>& get_fused_primitives_onednn() const { return _impl_params->fused_desc_onednn; }
```

Comment:
Where it's used or gonna be used? Why we need it in kernel_impl_params?

Reply:
They are also modified to use `kernel_impl_param` instead of `program_node` in inference time. `get_node_input_layout()` and `get_fused_primitives_onednn()` are added to be used in `configure_post_ops_arguments()` of `onednn_impl`. In the case of `get_node_output_layout()`, I think that there is no problem using the one stored in `_impl_params` instead of `_node_output_layout`.

Anyway, I thought that the `kernel_impl_params` was introduced to remove dependency on `program_node` in inference time. Please correct me if I'm wrong. Thank you.

Reply:
I just don't want kernel_impl_params structure being scrapyard for any primitive parameters. To my mind, the structure should be used for 2 purposes: implementation selection and shape propagation. So if a parameter is needed for one of those, we can add it to the structure. Otherwise, it should be a part of primitive_inst or its subclass. So please revise the parameters introduced by this PR and ensure that those are really needed in kernel_impl_params

Reply:
Following your guide, I relocated all parameters introduced by this PR from kernel_impl_params to primitive_inst except fused_desc_onednn. Thank you.